### ISSUE TITLE
Trying to run Winetricks while another wine process is running renders Heroic unusable

### ISSUE THREAD

**Issue by arielj** [Heroic-Games-Launcher/HeroicGamesLauncher#2441]
### Describe the bug

When opening Winetricks, it waits for other wine processes to stop before showing the screen. While that's happening, Heroic shows an empty log modal to show the output of winetricks. This modal cannot be closed, it only closes when winetricks is closed, so, if winetricks never loads, it cannot be closed so the modal blocks any UI interaction.

### Add logs

```text
Nothing relevant
```

### Steps to reproduce

1. Start a game
2. Without closing the game, go to the settings and run Winetricks
3. The modal shows up and you get stuck there
4. Close the game
5. Winetricks shows up and you can close it to close the modal

Running winetricks while a game is running sounds like an unrealistic situation, but a wine process can be running in the background for other reasons (like a game that failed to exit properly) and it may not be easy for casual users to find the wine processes to kill them.

### Expected behavior

Clicking the `X` in the log modal should abort the winetricks execution, maybe with a confirmation dialog to prevent closing it by mistake. I think it's ok that clicking the backdrop doesn't close the modal.

### Heroic Version

Latest Stable

### System Information

- OS [e. g. "Ubuntu"]: Linux Mint

**Comment by Etaash-mathamsetty** [comment]
winetricks should be able to open even with other processes open in the wine prefix, this is really strange, maybe we aren't setting the fsync and esync vars correctly since I am seeing a lot fsync/esync related errors which are saying that the prefix was created without esync/fsync

**Comment by Etaash-mathamsetty** [comment]
yeah here it is, esync and fsync are not set correctly
```
      const linuxEnvs = {
        ...process.env,
        WINEPREFIX: winePrefix,
        PATH: `${winepath}:${process.env.PATH}`
      }
```

**Comment by Etaash-mathamsetty** [comment]
not sure how to fix this issue since that requires access to the gameSettings, but I will try to figure out some way

**Comment by Nocccer** [comment]
Just add a close button to the modal. I think the modal has some property for that already

**Comment by arielj** [comment]
We have to make sure that closing the modal aborts the winetrick start too, so we don't end up with a winetricks trying to open in the background and eventually open out of nowhere haha

**Comment by flavioislima** [comment]
I think first thing is to disable the settings button when the game is running. Makes no sense since changing the settings wont affect it until it is launched again. After that we should remove the wineboot part of the dxvk/vkd3d removal since it is also creating more wine processes. With that, I don't think we need to care about winetricks.
OFC there is still the case where a person open the settings button for another game 🤔

**Comment by Etaash-mathamsetty** [comment]
partially fixed by #2444

**Comment by Etaash-mathamsetty** [comment]
got fixed sometime between 2023 and 2024
